src/storage/store.js

```diff
@@ -1,5 +1,5 @@
 import { Validator } from "jsonschema";
-import { merge } from "lodash";
+import merge from "lodash/merge";
```

Comment:
is `this.state` flat / only one level deep?

if so, [the `merge` call below](https://github.com/mozilla/hubs/pull/356/files#diff-0a39633babd056d1b5060747099dd5b6R72) could be a `Object.assign` call.

if it's possible for `this.state` to be deep, then, yeah, stick with `lodash/merge` :+1:

Reply:
It looks like it's two levels deep right now and may change so yeah we probably want to use the lodash merge.

Reply:
sounds good, go for it - tested, works